Solutions/Digital Shadows/Analytic Rules/Digital_Shadows_incident_creation_include.yaml

```diff
@@ -0,0 +1,84 @@
+id: ede3071d-9317-45f9-b36c-6a6effee5294
```

Comment:
Please use the same suggestions that we have for the previous query to make requisite changes.